src/en/wip-systems.md

```diff
@@ -30,10 +30,10 @@ A Juju Environment System (JES), also sometimes shortened to 'Juju System',
 describes the environment that runs and manages the Juju API servers and the
 underlying database.
 
-This initial environment is also called the 'system environment' 
-(or 'bootstrap environment'), and is what is
-created when the bootstrap command is used.  This System environment is a
-normal Juju environment that just happens to have machines that manage Juju.
+This initial environment is also called the 'system environment' (or
+'bootstrap environment'), and is created when the bootstrap command is used.
+This System environment is a normal Juju environment that just happens to have
+machines that manage Juju.
 
```

Comment:
I think we need to figure out the terms and stick with them in this document to allow others to think about things properly.

Thing 1: the initially bootstrapped Juju that then hosts and manages subsequent environments.  

Thing 2: The result of 'juju system create-environment'.  It is the equivalent of a normal 'juju bootstrap' but lives on top of Thing 1.  You can create as multiples of these Things.

So it sounds like thus far we've been calling Thing 1 "System" and Thing 2 things "Environments".

User management happens at the System level but users can share environments among themselves.  I think my suggestion to show the full workflow (annotated) is the right way to describe what's happening.  Perhaps with a block diagram.